### DISCUSSION TITLE
Can Infinity be used with the Boomtable panel?

### DISCUSSION THREAD

**Discussion by kgroshert** [grafana/grafana-infinity-datasource#294] [Q&A] [answered]
I have CSV-data with information about database-states and I'd like to color the cells differently if "ONLINE" or "OFFLINE" is somewhere contained in the column text. With the Grafana table panel there seems to be a years-long feature-request for this but it is not implemented yet.
However, Boomtable could do it as far as I've seen. However I struggle getting it to work, I guess because these are not timeseries-data. I just have the columns "db_name" and "db_status"... 
Can it be done? With a transform maybe? 🙄

**Comment by kgroshert** [comment, marked as answer]
Ah, it can indeed be done with the Grafana built-in table-panel! In the value-mappings editor there is only a precise text match. But if you add another mapping you can select "Regex" - which is what I needed.

**Reply by yesoreyeram** [reply]
Hiya.. sorry for the delay. Glad you found what you needed. 
Infinity datasource is not compatible with boom table. (Boom table was designed for legacy grafana data formats)
